kubectl-cloudflow/cmd/verify.go

```diff
@@ -0,0 +1,170 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/go-akka/configuration"
+	"github.com/lightbend/cloudflow/kubectl-cloudflow/domain"
+	"github.com/lightbend/cloudflow/kubectl-cloudflow/util"
+	"github.com/lightbend/cloudflow/kubectl-cloudflow/verify"
+	"github.com/spf13/cobra"
+)
+
+type verifyBlueprintCMD struct {
+	cmd *cobra.Command
+}
+
+func init() {
+	verifyBlueprintCMD := &verifyBlueprintCMD{}
+	verifyBlueprintCMD.cmd = &cobra.Command{
+		Use:     "verify",
+		Short:   "Verifies the blueprint of a Cloudflow Application.",
+		Example: "kubectl cloudflow verify <blueprint-file-name>",
+		Run:     verifyBlueprintCMD.verifyImpl,
+		Args:    validateVerifyCmdArgs,
+	}
+	rootCmd.AddCommand(verifyBlueprintCMD.cmd)
+}
+
+// this map is constructed entirely from blueprint
+func getImageRefsFromConfig(config *configuration.Config) map[string]domain.ImageReference {
+	// get the images from the blueprint
+	images := config.GetNode("blueprint.images").GetObject().Items()
+
+	var imageKeyVals = make(map[string]domain.ImageReference)
+
+	for imageKey, imageRef := range images {
+		ref, err := verify.ParseImageReference(imageRef.GetString())
+		if err != nil {
+			util.LogAndExit(err.Error())
+		}
+		imageKeyVals[imageKey] = *ref
+	}
+	return imageKeyVals
+}
+
+func getFilePath(pathToBlueprint string) (string, error) {
```

Comment:
Is it documented in verify that you can specify a file by URL or by local file? we need to make this consistent, also in deploy, or any file path that the CLI uses.

Reply:
Maybe rename this in a way that it is clear that it returns the filename if it is a local file, otherwise downloads the file from http if it is a url. It's slightly odd tbh.

Reply:
Thinking about it a little longer I think we should only support local files right now. This can be added later, and then we can look at all the corner cases..

Reply:
@RayRoestenburg 
a) I can remove the url support but I find that it adds value even if it does not cover all cases (which are?)
b) Making something consistent shouldnt be hard the functionality is there for the blueprint (not required for any file at the moment). 
Anyway @debasishg what do you think?